src/sw-template.js

```diff
@@ -0,0 +1,97 @@
+if ('function' === typeof importScripts) {
+    importScripts(
+      'https://storage.googleapis.com/workbox-cdn/releases/3.5.0/workbox-sw.js'
+    );
+    
+    /* global workbox */
+    if (workbox) {
+      
+      console.log('Workbox for SFF is loaded');
```

Comment:
Should we only log this in debug mode?

Reply:
This was simply removed 7b1fa7f24f15947b5e0f6b4fda731c57e89f3a87